nml/actions/action0properties.py

```diff
@@ -818,6 +818,17 @@ def station_tile_flags(value):
     ]
 
 
+def array_for_station_properties(prop_num, list_type="Flag"):
+    def __call__(value):
+        if not isinstance(value, Array):
+            raise generic.ScriptError(list_type + " list must be an array", value.pos)
```

Comment:
generally speaking i'd say use f-strings instead of concatenation. But I suspect nml doesn't really use them because it's too old...